### PR TITLE
clean up metatag tests

### PR DESCRIPTION
**Pull request by ngokevin**
**Description:** Follow up to https://github.com/aframevr/aframe/pull/1112 for @cvan

**Changes proposed:**
- Move metaTag tests to different file from scene.test.js
- Remove rendering from metaTag tests
- Simplify setup for metaTag tests

### CODE REVIEW

**Comment by cvan** [comment]
wooo, thanks, @ngokevin! appreciate it

File: tests/core/scene/meta-tags.test.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+var initMetaTags = require('core/scene/metaTags').inject;
+var metaTags = require('core/scene/metaTags');
+
+suite('metaTags', function () {
+  setup(function (done) {
+    var el = this.el = helpers.entityFactory();
+    var i;
+    var metaTagEls = document.querySelectorAll('meta');
+    el.addEventListener('loaded', function () {
+      for (i = 0; i < metaTagEls.length; i++) {
```
Comment (cvan) [inline tests/core/scene/meta-tags.test.js]: why's this have to happen after `loaded`?

File: tests/core/scene/meta-tags.test.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    var sceneEl = this.el.sceneEl;
+    var webAppCapableMetaTag;
+
+    initMetaTags(sceneEl);
+
+    webAppCapableMetaTag = document.querySelector('meta[name="web-app-capable"]');
+    appleMobileWebAppCapableMetaTag = document.querySelector(
+      'meta[name="apple-mobile-web-app-capable"]');
+
+    assert.equal(document.querySelector('meta[name="viewport"]').content,
```
Comment (cvan) [inline tests/core/scene/meta-tags.test.js]: to be consistent, we should probably have a `var viewportMetaTag` and assert that exists first

**Comment by ngokevin** [comment]
r?

**Pull request merged**